compat/src/render.js

```diff
@@ -50,10 +50,18 @@ export function hydrate(vnode, parent, callback) {
 let oldEventHook = options.event;
 options.event = e => {
 	if (oldEventHook) e = oldEventHook(e);
-	e.persist = () => {};
-	e.isDefaultPrevented = () => e.defaultPrevented;
-	e.isPropagationStopped = () => e.stopPropagation;
-	return (e.nativeEvent = e);
+	if (e.nativeEvent === undefined) {
+		e.persist = () => {};
+		e.isDefaultPrevented = () => e.defaultPrevented;
+		const stopPropagation = e.stopPropagation;
+		event.stopPropagation = function() {
+			stopPropagation.call(this);
```

Comment:
I merged the last one too fast, I think just keeping the old code and adding some local variable would be better since this is going to add a lot of size

Reply:
IMHO,
I think it is better that `event.isDefaultPrevented()` and `event.isPropagationStopped()` are not supported, since it seems they has a little needs .